samples/XCT.Sample/ViewModels/Converters/DoubleToIntConverterViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+namespace Xamarin.CommunityToolkit.Sample.ViewModels.Converters
+{
+	public class DoubleToIntConverterViewModel : BaseViewModel
+	{
+		double index;
+		public double Input
+		{
+			get => index;
+			set
+			{
+				index = value;
+				OnPropertyChanged(nameof(Input));
```

Comment:
We don't need the `nameof` here, you can change this and others?
```suggestion
				OnPropertyChanged();
```